Geometry_Engine/Create/NTree.cs

```diff
@@ -0,0 +1,47 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2020, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Geometry;
+using BH.oM.Geometry.CoordinateSystem;
+using System;
+using System.Linq;
+using System.ComponentModel;
+using BH.oM.Data.Collections;
+using System.Collections.Generic;
+
+namespace BH.Engine.Geometry
+{
+    public static partial class Create
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        public static NTree<T> NTree<T>(IEnumerable<T> geometry, int split = 16) where T : IGeometry
+        {
+            return Data.Create.NTree(geometry.Select(x => new Leaf<T>() { Bounds = x.IBounds().NBound(), Item = x }), split);
+        }
```

Comment:
Managed to avoid the before mentioned bug where one couldn't copy-paste one of the query methods connected to this by using the more explicit Create method in the Data_Engine instead.

Strongly suspect that it's the `where T : IGeometry` which is messing things up since I think that is the only difference.